src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -96,4 +96,27 @@ private function addPurgeAuthHeader(array $headers)
 
         return $headers;
     }
+
+    /**
+     * Get amount of tags per header, adapted from FOSHttpCache 2.x.
+     *
+     * @param array $tags
+     *
+     * @return int
+     */
+    private function determineTagsPerHeader(array $tags)
+    {
+        if (!$this->configResolver->getParameter('http_cache.varnish_bulk_purge')) {
+            return 1;
+        }
+
+        if (mb_strlen(implode(' ', $tags)) < self::DEFAULT_HEADER_LENGTH) {
```

Comment:
Nitpick: Could you please extract const representing tags separator? It's hardcoded in few places ;-)